doc/Learn/Navigation/Advanced/HowTo-AdvancedPageNavigation.md

```diff
@@ -7,6 +7,14 @@ When using navigation, you may not want to allow the current page to remain in t
 
 Using backward/forward navigation in your app requires a degree of extra consideration. Users always expect the back/forward button to take them to a page that is still relevant, yet logically related to the current page and direction. This page contains several concise tutorials about how to implement navigation techniques that address these more advanced problems.
 
+## Step-by-steps
+
+> [!IMPORTANT]
+> This guide assumes you used the template wizard or `dotnet new unoapp` to create your solution. If not, it is recommended that you follow the [Creating an application with Uno.Extensions article](xref:Uno.Extensions.HowToGettingStarted) for creating an application from the template.
+
+> [!NOTE]
+> This guide uses predefined code created by the Uno Template using the `Recommended` preset, however it uses the `Mvvm` approach for the examples instead of `MVUX` defined in the `Recommended` preset.
```

Comment:
```suggestion
> This guide uses predefined code created by the Uno Template using the `Recommended` preset, however, it uses the `MVVM` approach for the examples instead of `MVUX` defined in the `Recommended` preset.
```

Reply:
@eriklimakc, @nickrandolph  wondering if we could maybe have both versions for this guide? MVVM + MVUX (+ create links to refer to MVVM and MVUX definitions/details)?

Reply:
Huum, not sure if it's really needed. I understand it's nice to have both. What would change is basically is Model vs ViewModel naming. 

Maybe just add a link to MVUX so that people understand deeply what changes from MVVM to MVUX?

Reply:
I will let you check with @nickrandolph what makes more sense at the end here

Reply:
I think for now the notice in the docs is sufficient